GooglePlayInstant/Editor/QuickDeploy/AccessTokenGetter.cs

```diff
@@ -187,5 +235,54 @@ public class GcpAccessToken
             /// </summary>
             public int expires_in;
         }
+
+        /// <summary>
+        /// Holds utility values and methods that enable to hold and retrieve the values of the current access and
+        /// refresh tokens, as well as to track whether the current token has expired or not. 
+        /// </summary>
+        [Serializable]
+        public static class AccessToken
```

Comment:
Suggestion: the class and its fields should not be static. Instead treat this as an immutable object, store an instance of it as a static field in the parent class, and create a new object when creating or refreshing the token.

You could continue to have an AccessToken property as before, with the default value being one that has default values such that IsValid will return false.

After these changes are made, it would be easier to unit test this class.

Reply:
Resolved. Still made refresh token static since it is shared by the entire class.

Added some unit tests as well.